src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const div = document.createElement('div');
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve(12);
+  });
+});
+
+promise1
+  .then(() => {
+    body.append(div);
+    div.classList = 'message';
+    div.textContent = 'Promise was resolved!';
```

Comment:
It would be great to create a separate function for showing a message as it's almost the same for success and reject.
Also, you can get an argument of resolve\reject as then\catch parameter